Filter/StringFilter.php

```diff
@@ -25,28 +25,39 @@ public function filter(ProxyQueryInterface $proxyQuery, $alias, $field, $data)
             return;
         }
 
-        $data['value'] = trim($data['value']);
+        $value = $data['value'] = trim($data['value']);
```

Comment:
why update $data['value'] here? you never read from it below. i am ok moving this to $value